runtime/src/accounts_db.rs

```diff
@@ -260,6 +263,17 @@ impl GenerateIndexTimings {
                 i64
             ),
             ("index_flush_us", self.index_flush_us as i64, i64),
+            (
+                "total_not_rent_exempt_with_duplicates",
```

Comment:
I find the "not rent exempt" to be another double-negative. Maybe this could be "rent paying" instead?

I'm guessing what goes in here will be how we all end up talking about these numbers, so I'd love to have that be something that's easier for me too 🙃

(and same for the other uses of "non rent exempt")

Reply:
it is double. we'll see what we find useful. this is what was asked for as far as I could tell.